crm/main.py

```diff
@@ -68,12 +70,13 @@ def show_resources(self, resource):
         header = ("Resource Name", "Used Count", "Available Count")
         data = []
 
-        if resource == 'all':
-            for res in ["ipv4_route", "ipv6_route", "ipv4_nexthop", "ipv6_nexthop", "ipv4_neighbor", "ipv6_neighbor",
-                        "nexthop_group_member", "nexthop_group", "fdb_entry"]:
-                data.append([res, crm_stats['crm_stats_' + res + "_used"], crm_stats['crm_stats_' + res + "_available"]])
-        else:
-            data.append([resource, crm_stats['crm_stats_' + resource + "_used"], crm_stats['crm_stats_' + resource + "_available"]])
+        if crm_stats:
```

Comment:
The _else_ part has to display some message stating crm_stats is _not_ready_ since it would be populated only after the polling interval